### PR TITLE
Evaluation

### PR DESCRIPTION
**Pull request by andreasalstrup**
We can now evaluate and compare multiple trained models.

Implemented in `evaluation.py` and called in `train.ipynb`: Models in the `model/models/` directory will be evaluated and compared.

### CODE REVIEW

File: train.ipynb [hunk 1]
```diff
@@ -315,71 +316,83 @@
...
     "time_start = timer()\n",
     "\n",
+    "# Setup/Load checkpoint\n",
     "checkpoint_epoch = -1\n",
     "CHECKPOINT = 'model/models/checkpoint.pt'\n",
     "CHECKPOINT_BACKUP = 'model/models/checkpoint_backup.pt'\n",
     "\n",
     "if path.exists(CHECKPOINT):\n",
-    "    (model, optimizer, checkpoint_epoch, show_training, show_testing, train_dataloader, test_dataloader) = loadCheckpoint(CHECKPOINT, model, optimizer)\n",
+    "    (model, optimizer, checkpoint_epoch, show_Training, show_Testing, TRAIN_DATALOADER, TEST_DATALOADER) = loadCheckpoint(CHECKPOINT, model, optimizer)\n",
```
Comment (malgreen) [inline train.ipynb]: It should still be show_training, not combined snake- + camel case

**Comment by malgreen** [comment]
describe changes and why you made them

**Comment by andreasalstrup** [comment]
> describe changes and why you made them

Fixed

**Review by malgreen — APPROVED** [review]
alright......... lgtm

**Pull request merged**